client/src/lmclient.cpp

```diff
@@ -118,6 +120,8 @@ int main(int argc, char* argv[])
             if (i >= argc) { usage(); return 1; }
             refChord = string(argv[i]);
             minArgCnt++;
+        } else if ((strcmp(argv[i], "--tplsdump") == 0)) {
+            tplsDump = true;
```

Comment:
```suggestion
            dumpTemplates();
            return 0;
```
Just like we do for  `printScales()`, isn't that simpler ?